root/layout/components/BottomMenu.js

```diff
@@ -170,7 +170,7 @@ const ProductsMenu = () => (
         <a href="/doc/Developer_Resources">{l('Developer Resources')}</a>
       </li>
       <li>
-        <a href="/doc/XML_Web_Service">{l('XML Web Service')}</a>
+        <a href="/doc/MusicBrainz_API">{l('JSON/XML API')}</a>
```

Comment:
Maybe name it “MusicBrainz API” like you did for the target page itself and [above](https://github.com/metabrainz/musicbrainz-server/pull/1592/commits/481001db2e563bc46ae1000331862b21e2cbd514#diff-4c3d028cef4917d20675baed1fb0ce28R125):

```suggestion
        <a href="/doc/MusicBrainz_API">{l('MusicBrainz API')}</a>
```

Reply:
+1 to that. I was just about to open a PR to rename this menu link to MusicBrainz API, but found MBS-10941.